Sources/PT.PM.Common/LanguageUtils.cs

```diff
@@ -102,7 +128,7 @@ public static Language[] GetLanguagesByExtension(string extension)
 
         public static bool HasAntlrParser(this Language language) => LanguageInfos[language].HasAntlrParser;
 
-        public static bool IsParserExists(this Language language) => LanguagesWithParser.Contains(language);
+        public static bool IsParserExists(this Language language) => LanguagesWithParser.Contains(language) || language.IsSerialization();
```

Comment:
Why not just add `SerializationLanguages` to `LanguagesWithParser`?

Reply:
Because serialization languages (json, msgpack) are being processed in another way and their parsers are not being registered in [RegisterAllParsersAndCovnerters method](https://github.com/PositiveTechnologies/PT.PM/blob/dev/Sources/PT.PM/Utils.cs#L27). That's why `LanguagesWithParser` does not contain parsers for Json and MessagePack.

But I'll think whether I can refactor this moment.

Reply:
Finally, I decided to rename this method to `IsParserExistsOrSerialized`.

Reply:
`IsParserExistsOrSerialized` reads like "is parser exists or the parser is serialized" which isn't really accurate. :)
What's the reason they are combined? What's the common processing?

Reply:
You are right. Correct identifier: `IsParserExistsOrSerializing`. This means "is a parser for language exists or is language serializing`.